client/core/core_test.go

```diff
@@ -48,36 +42,55 @@ import (
 	"golang.org/x/text/message"
 )
 
+func init() {
+	asset.Register(tUTXOAssetA.ID, &tDriver{
+		decodedCoinID: tUTXOAssetA.Symbol,
```

Comment:
What do you think about `tUTXOAssetA.Symbol + "-coin"` so it reads like `dcr-coin` in the logs or wherever.

Reply:
Done.